SDL/Makefile

```diff
@@ -21,13 +21,13 @@ CXX                  = nvcc
 CXXFLAGS             =  -g --compiler-options -Wall --compiler-options -Wshadow --compiler-options -Woverloaded-virtual --compiler-options -fPIC --compiler-options -fopenmp -dc -lineinfo --ptxas-options=-v --cudart shared -arch=compute_52 -I/mnt/data1/dsr/cub 
 LD                   = nvcc 
 SOFLAGS              = -g -shared --compiler-options -fPIC --cudart shared -arch=compute_52
-PRINTFLAG            = -DAddObjects -DT4FromT3 #-DWarnings
+PRINTFLAG            = -DAddObjects -DDO_QUINTUPLET #-DT4FromT3 -DWarnings
```

Comment:
-DT4FromT3 should be on by default (at least when running T4s)

Reply:
When running T4 side by side with T5, This is useful to have a comparison of times.